.gitignore

```
@@ -34,8 +34,41 @@
 # Web related
 lib/generated_plugin_registrant.dart
 
+# iOS/XCode related
+**/ios/**/*.mode1v3
+**/ios/**/*.mode2v3
+**/ios/**/*.moved-aside
+**/ios/**/*.pbxuser
+**/ios/**/*.perspectivev3
+**/ios/**/*sync/
+**/ios/**/.sconsign.dblite
+**/ios/**/.tags*
+**/ios/**/.vagrant/
+**/ios/**/DerivedData/
+**/ios/**/Icon?
+**/ios/**/Pods/
+**/ios/**/.symlinks/
+**/ios/**/profile
+**/ios/**/xcuserdata
+**/ios/.generated/
+**/ios/Flutter/.last_build_id
+**/ios/Flutter/App.framework
+**/ios/Flutter/Flutter.framework
+**/ios/Flutter/Flutter.podspec
+**/ios/Flutter/Generated.xcconfig
+**/ios/Flutter/app.flx
+**/ios/Flutter/app.zip
+**/ios/Flutter/flutter_assets/
+**/ios/Flutter/flutter_export_environment.sh
+**/ios/ServiceDefinitions.json
+**/ios/Runner/GeneratedPluginRegistrant.*
```

I don't think we need the leading double stars :)
```suggestion
/ios/**/*.mode1v3
/ios/**/*.mode2v3
/ios/**/*.moved-aside
/ios/**/*.pbxuser
/ios/**/*.perspectivev3
/ios/**/*sync/
/ios/**/.sconsign.dblite
/ios/**/.tags*
/ios/**/.vagrant/
/ios/**/DerivedData/
/ios/**/Icon?
/ios/**/Pods/
/ios/**/.symlinks/
/ios/**/profile
/ios/**/xcuserdata
/ios/.generated/
/ios/Flutter/.last_build_id
/ios/Flutter/App.framework
/ios/Flutter/Flutter.framework
/ios/Flutter/Flutter.podspec
/ios/Flutter/Generated.xcconfig
/ios/Flutter/app.flx
/ios/Flutter/app.zip
/ios/Flutter/flutter_assets/
/ios/Flutter/flutter_export_environment.sh
/ios/ServiceDefinitions.json
/ios/Runner/GeneratedPluginRegistrant.*
```

---

Got this from https://github.com/flutter/flutter/blob/master/.gitignore 
But in this path tree its not needed.